ompi/mca/hook/comm_method/hook_comm_method_component.c

```diff
@@ -113,61 +134,70 @@ static int ompi_hook_comm_method_component_register(void)
     /*
      * If the component is active for mpi_init / mpi_finalize
      */
+    // hook_comm_method_enable_mpi_init
     mca_hook_comm_method_enable_mpi_init = false;
     (void) mca_base_component_var_register(&mca_hook_comm_method_component.hookm_version, "enable_mpi_init",
-                                           "Enable comm_method behavior on mpi_init",
+                                           "Enable the communication protocol report when MPI_INIT is invoked",
                                            MCA_BASE_VAR_TYPE_BOOL, NULL,
                                            0, 0,
                                            OPAL_INFO_LVL_3,
                                            MCA_BASE_VAR_SCOPE_READONLY,
                                            &mca_hook_comm_method_enable_mpi_init);
 
+    // hook_comm_method_enable_mpi_finalize
     mca_hook_comm_method_enable_mpi_finalize = false;
     (void) mca_base_component_var_register(&mca_hook_comm_method_component.hookm_version, "enable_mpi_finalize",
-                                           "Enable comm_method behavior on mpi_finalize",
+                                           "Enable the communication protocol report when MPI_FINALIZE is invoked",
```

Comment:
Just curious -- why do we have `hook_comm_method_enable_mpi_init`, `hook_comm_method_enable_mpi_finalize`, _and_ `ompi_comm_method`?  They seem to be redundant.

Should we pick one set or the other?  I kinda like the `ompi_comm_method` enum flags, but have no strong preference here.

Reply:
That's a fair point. I'll take out the other two options, and leave the `ompi_comm_method`.

Reply:
Fixed in latest push.